pages/[lng]/volunteers/index.js

```diff
@@ -1,13 +1,98 @@
 import Layout from '../../../components/Layout';
 import { charityAPI } from '../../../clients';
-import { Soon } from '../../../components/Soon';
+import { VolunteersSection } from '../../../components/VolunteersSection';
+import { VolunteeringBanner } from '../../../components/VolunteeringBanner';
```

Comment:
Don't limit/tie this banner to Volunteers domain, It can be used in any other page with any other content, name it more generically, similar to what we did with `FeaturedBanner` in  homepage. so `FeaturedBannerSecondary` could be more abstract.